README.md

```diff
@@ -60,6 +60,11 @@ jwerl:verify(Jwt, rs512, PublcPem).
 Jwt = jwerl:sign([{name, <<"bob">>}], es256, PrivtPem).
 jwerl:verify(Jwt, es256, PublcPem).
 
+% Compatibility
+%   - sign/verify the signature with raw format instead of DER
+%   - it is necessary to compatible with nodejs or other platforms
+Jwt = jwerl:sign([{name, <<"bob">>}], es256, PrivtPem, #{raw => true}).
+jwerl:verify(Jwt, es256, PublcPem, Claims, #{raw => true}).
```

Comment:
Please do not modify the README.md directly. Indeed, it is generated by `edown`. You must add this in the `overview.edoc` file.